velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -529,10 +529,19 @@ TypePtr ReaderBase::convertType(
             VELOX_FAIL(
                 "UTF8 converted type can only be set for thrift::Type::(FIXED_LEN_)BYTE_ARRAY");
         }
+
+      case thrift::ConvertedType::ENUM:
```

Comment:
Would it be possible to add a test that exercises this code path?

Reply:
Sure. I will try generating one example file for it.

Reply:
Hi @mbasmanova I tried creating an example file as the original data from customer is too large but found schema changes after re-wrote by Spark, which has a new convertedType `UTF8`.
<img width="266" alt="image" src="https://github.com/facebookincubator/velox/assets/11849056/a81025cd-f9c2-4507-a374-7e80e1127f23">
do you think it's okay without an UT?

Reply:
UTF8 won't hit ENUM code path, will it?

Reply:
No. It is another already existing path.

Reply:
If UT8 path is not covered, then let's add that test. Still we need a test that covers ENUM path.